WebAPI/Repository/IBetRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using WebAPI.Model.Entities;
+
+namespace WebAPI.Repository
+{
+    public interface IBetRepository
+    {
+        List<Bet> GetBets();
+        Bet GetBetById(int ID);
+        void SaveBet(Bet bet);
+        void DeleteBet(int ID);
```

Comment:
Can we try to add generic Repository? 
https://codingblast.com/entity-framework-core-generic-repository/